front/components/assistant/conversation/InputBar.tsx

```diff
@@ -609,6 +614,30 @@ export function AssistantInputBar({
                         }}
                       />
                     ))}
+                    <div className="flex flex-row justify-between border-t border-structure-100 px-3 py-2">
+                      <Button
```

Comment:
There is no way to use `<Link>` here ?

Reply:
We can (and should) wrap the Button in a Link :+1: